content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md

```diff
@@ -51,6 +51,18 @@ the `.items[*]` portion of the path should be omitted because a single
 Pod is returned instead of a list of items.
 {{< /note >}}
 
+## List all images of the workload resources
+
+Listing Pods may yield different results based on the Pods defined lifecycle.
+
+The following example lists all the images possible run in the cluster, except 
+for Orphaned Pods created directly without using workload resources. 
```

Comment:
```suggestion
for {{< glossary_tooltip text="static" term_id="static-pod" >}} or standalone
Pods (standalone means Pods that you created directly without using workload resources).
```
What if someone is using a custom controller that manages Pods, though?

Reply:
I assume if a custom controller manages Pods directly it would be a Static Pod